src/index.js

```diff
@@ -226,6 +228,7 @@ exports.init = (Implementation) => {
         configStore.lianaOptions,
       ).perform();
     })
+    .then(() => new HealthCheckRoute(app, configStore.lianaOptions).perform())
```

Comment:
Why do we mount a route/middleware twice?

Reply:
bad mistake!